### PR TITLE
Added RDesktop

### PR DESCRIPTION
**Pull request by ItsNoss**
# Summary

I added RDesktop to the template file and created a stack template. I also added the icon for RDesktop.

### CODE REVIEW

**Comment by ItsNoss** [comment]
This is only working on arm64.

**Comment by ItsNoss** [comment]
I updated it to work with arm32. Although slower, it does work.

**Comment by rmiddle** [comment]
I don't see the purpose of stack/rdesktop.yml  ? Please remove it from your PR, and then I will accept this change.

File: template/portainer-v2-arm64.json [hunk 1]
```diff
@@ -5830,6 +5830,45 @@
...
+					"name": "PGID"
+				},
+                {
+					"default": "America/New_York",
+                    "label": "TZ",
+                    "name": "TZ"
+                }
+			],
+			"image": "linuxserver/rdesktop:arm64v8-latest",
+			"logo": "https://raw.githubusercontent.com/ItsNoss/pi-hosted/master/images/rdesktop.png",
```
Comment (blumberg) [inline template/portainer-v2-arm64.json:5857]: Point to correct repo:

```suggestion
			"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/rdesktop.png",
```

File: template/portainer-v2-arm32.json [hunk 2]
```diff
@@ -5551,6 +5551,45 @@
...
+					"name": "PGID"
+				},
+                {
+					"default": "America/New_York",
+                    "label": "TZ",
+                    "name": "TZ"
+                }
+			],
+			"image": "linuxserver/rdesktop:latest",
+			"logo": "https://raw.githubusercontent.com/ItsNoss/pi-hosted/master/images/rdesktop.png",
```
Comment (blumberg) [inline template/portainer-v2-arm32.json:5578]: Point to correct repo
```suggestion
			"logo": "https://raw.githubusercontent.com/pi-hosted/pi-hosted/master/images/rdesktop.png",
```

**Review by blumberg — COMMENTED** [review]
`Kanboard_icon.png` file does not belong to this app

`stack/rdesktop.yml` is not needed as you're not using Template type 3 (stacks)

**Comment by ItsNoss** [comment]
I have removed the RDesktop Stack and Kanboard Icon and updated the repo that the icon points to

**Pull request merged**